### PR TITLE
Design WebsiteInfo component

### LINKED ISSUE #8: Design WebsiteInfo component

### PR DESCRIPTION
**Pull request by anthonyminyungi**
- It closes #8

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { LoginPage, RegisterPage } from './components';
 export default function App() {
   const dummyData = {
     title: 'Naver',
-    favicon: '',
+    favicon: 'https://s.pstatic.net/static/www/mobile/edit/2016/0705/mobile_212852414260.png',
```
Comment (anthonyminyungi) [inline src/App.tsx:8]: Favicon URL of [naver](www.naver.com)
스크린샷 2020-08-29 오후 10 37 10

File: src/components/common/info/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    return (
+        <Container className={classes.container}>
+            <Avatar className={classes.icon} alt={title} src={favicon} />
+            <Typography className={classes.title}>
+                {title}
+            </Typography>
+        </Container>
+    );
+};
```
Comment (anthonyminyungi) [inline src/components/common/info/index.tsx:36]: An example of appearance.
스크린샷 2020-08-29 오후 10 40 00

**Review by anthonyminyungi — COMMENTED** [review]
Self-review

File: src/App.tsx (hunk 1 above)
Comment (HyeockJinKim) [reply src/App.tsx:8]: It seems that dummy data should be written as test code later.

**Review by HyeockJinKim — APPROVED** [review]
LGTM.

**Pull request merged**